File: 프로그래머스/lv0/120824. 짝수 홀수 개수/짝수 홀수 개수.js
function solution(num_list) {
    let newArr = [];
    let num1 = 0;
    let num2 = 0;
    num_list.forEach( (num) => {
        if(num%2 == 0) {
            num1 += 1;
        }else{
        num2 += 1;
        }
    });
    newArr.push(num1);
    newArr.push(num2);
    return newArr;
}

File: 프로그래머스/lv0/120824. 짝수 홀수 개수/짝수 홀수 개수.java
class Solution {
    public int[] solution(int[] num_list) {
        int a = 0;
        int b = 0;
        int[] newArr = new int[2];
        for(int i=0; i<num_list.length; i++) {
            if(num_list[i]%2 == 0) {
                a += 1;
            }else {
                b += 1;
            }
        }
        newArr[0] = a;
        newArr[1] = b;
        return newArr;
    }
}